test/Feature/SingularSpec.hs

```diff
@@ -144,9 +144,8 @@ spec =
 
         -- the rows should not exist, either
```

Comment:
I think this comment was incorrectly copied and pasted from earlier. As you point out, it's not accurate here. The rows were not deleted so they *should* still exist.